### PR TITLE
Display search result

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Display search resultㄴ after pressing enter.
- Filter and sort search results. 

## QA

- Go to  https://canonical-com-575.demos.haus/careers/all
- e.g. Search "cloud" and press enter
- Check the job list displays properly 
- Check filter and sort works with the search result

## Issue / Card

Fixes #[5701](https://github.com/canonical-web-and-design/web-squad/issues/5701)

### CODE REVIEW

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
This looks good, but doesn't address a point @anthonydillon made in the previous PR - if there are no results, we should inform the user, like we do on live.

See:
https://canonical.com/careers/finance?filter=home-based&location=all#available-roles
vs
https://canonical-com-575.demos.haus/careers/all?search=fast

**Comment by minkyngkm** [comment]
Thanks @sowasred2012. I forgot about that. It's fixed now 👍

File: templates/careers/base.html [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 {% block extra_metatags %}
   <link rel="alternate" type="application/rss+xml"  href="/careers/feed" title="All job roles | Canonical Careers">
-  <script src="{{ versioned_static('js/modules/fuse/fuse.js') }}"></script>
```
Comment (sowasred2012) [inline templates/careers/base.html:5]: Not sure I understand why this was moved to base_index, won't that mean this script will be loaded on pages that don't use fuse?

**Review by sowasred2012 — APPROVED** [review]
Nice! LGTM

**Pull request merged**